source/_integrations/homekit.markdown

```diff
@@ -253,11 +253,16 @@ homekit:
               type: integer
               default: 1316
             video_codec:
-              description: Only for `camera` entities. FFmpeg video codec for transcoding. `copy` option reduces CPU load when video source already encoded with `H264` (MPEG4). `h264_omx` option is only available with custom FFmpeg builds and enables GPU Hardware acceleration on Raspberry Pi.
+              description: Only for `camera` entities. FFmpeg video codec for transcoding. `copy` option reduces CPU load when video source already encoded with `H264` (MPEG4). `h264_omx` option is only available with custom FFmpeg builds and enables GPU Hardware acceleration on Raspberry Pi. `h264_v4l2m2m` is also available for supported hardware, e.g., the Raspberry Pi, but does not require a custom build.
```

Comment:
`does not require a custom build`

Does it require a custom build?

Reply:
Well, it's auto-detected in FFmpeg's build configuration but it is enabled in Raspberry Pi OS (also in Debian/Ubuntu) and in the Home-Assistant Docker image. So I didn't need a custom build to use it

Reply:
I would drop the wording about a custom build in that case since they don't need to do anything in most cases

Reply:
I only mentioned that in contrast to (the now deprecated, I think) `h264_omx`. Would it be alright if I place m2m before omx? Then, perhaps, they'd be no need for the contrast and `h264_omx` would just remain the odd-man-out.

Reply:
That seems fine. I expect there aren't many people using the other codec since it requires a custom build and if they are there are no changes for them to make from this pr